### DISCUSSION TITLE
How to mutate all the keys that match a pattern?

### DISCUSSION THREAD

**Discussion by Rednegniw** [vercel/swr#1635] [Q&A]
For example, in our app we have keys like
- /projects
- /client/:id/projects
- /client/:id/projects?page=2&page_size=20

We use modals to edit individual projects, but its pretty hard to pass all the possible mutates. Is there a way where we can refetch every key that matches "projects"? perhaps even better, only rerender those keys that are currently rendered.

**Comment by shuding** [comment]
You can take a look at https://swr.vercel.app/docs/advanced/cache#mutate-multiple-keys-from-regex.
